bin/setup.bat

```diff
@@ -40,8 +40,9 @@ if not "%JAVA_TOOL_OPTIONS%" == "" (
 rem JAVA_OPTS is not a built-in JVM mechanism but some people think it is so we
 rem warn them that we are not observing the value of %JAVA_OPTS%
 if not "%JAVA_OPTS%" == "" (
-  echo|set /p="warning: ignoring JAVA_OPTS=%JAVA_OPTS%; "
+  echo warning: ignoring JAVA_OPTS=%JAVA_OPTS%;
   echo pass JVM parameters via LS_JAVA_OPTS
+  set JAVA_OPTS=
```

Comment:
we shouldn't need to also zero-out the `JAVA_OPTS` variable; it just isn't used anywhere downstream.

Reply:
exactly, this is not required because in `logstash.bat` we do not take into account what is set in `JAVA_OPTS`

Reply:
That's right, I think it's not necessary. I just wanted to do as same way is with JAVA_TOOL_OPTIONS

Reply:
yeah, `JAVA_TOOL_OPTIONS` is simply unset because not reused after but `JAVA_OPTS` is completely overwritten later on in logstash.bat.